docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1334,6 +1334,19 @@ Query Parameters:
 
 -   **Content-type** – Set to `"application/tar"`.
 -   **X-Registry-Config** – base64-encoded ConfigFile object
+-   **X-Docker-BuildArgs** – base64-encoded JSON map of string pairs for build-time
```

Comment:
No specific reason, I saw config being passed header so I used same approach for build-args as well. If there is a precedence or liking to do it through query-parameters I can look into changing this to query-parameters.